packages/plugin-javascript/index.js

```diff
@@ -92,6 +92,7 @@ export default {
         /\.es6$/,
         // CoffeeScript
         /\.coffee$/,
+        /\.vue?$/,
```

Comment:
The file extensions is always `.vue`, right? If so, you don't need the question mark otherwise the character in front of it will become optional.

Reply:
Will update.